src/aks-preview/azext_aks_preview/aks_app_commands/aksappcommands.py

```diff
@@ -0,0 +1,150 @@
+# pylint: disable=too-many-lines
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import sre_parse
+from typing import Dict, List
+import shutil
+import subprocess
+import requests
+import os
+import platform
+from pathlib import Path
+from knack.prompting import prompt_y_n
+
+
+def aks_draft_app_init(deployment_path: str,
+                       app_name: str,
+                       language: str,
+                       create_config: str,
+                       dockerfile_only: str,
+                       deployment_only: str) -> None:
+    file_path = _binary_pre_check()
+    if not file_path:
+        raise ValueError('Binary was NOT executed successfully')
+
+    arguments = _build_arguments(app_name, language, create_config, dockerfile_only, deployment_only)
+    run_successful = _run(file_path, deployment_path, arguments)
+    if run_successful:
```

Comment:
💡 📓 Quick note/observation: If not successful then I hope we are bubbling the error up.